src/cloudAdapter/cloudAdapter.ts

```diff
@@ -62,6 +62,7 @@ export interface CloudAdapter {
         input: GenezioCloudInput[],
         projectConfiguration: ProjectConfiguration,
         cloudAdapterOptions: CloudAdapterOptions,
+        stack: string[],
```

Comment:
Why is this a `string[]` instead of a `string`? How do you plan to use this feature in the future?

Reply:
My understanding is that a project will have a list of technologies defined in `stack`.
E.G. A NextJs project with an Express Backend will have the `stack: [nextjs, express]`.

For now, this is only implemented for NextJs projects, hence the stack is only `[nextjs]`.

Reply:
@andreia-oca is correct. That was the intention. Do you have any concerns, @costinsin ?

Reply:
Yes, I have. At the moment we either decide if the project is Next.js or classic deploy, so surely in at the moment there is no in between as @andreia-oca said. I am not sure how this will evolve in the future, but I am expecting to have one "stack" per project for now.

Reply:
I am also not sure how things will evolve, but I can see it happening that we might need a project to have multiple technologies. That's why I've implemented like this, in case we need it, we won't have to change anything.

However, if we won't need it, it will still work.